src/azure-cli/azure/cli/command_modules/appservice/_validators.py

```diff
@@ -219,6 +219,9 @@ def _validate_service_tag_format(cmd, namespace):
         name = namespace.name
         webapp = _generic_site_operation(cmd.cli_ctx, resource_group_name, name, 'get')
         service_tag_full_list = network_client.service_tags.list(webapp.location).values
+        if service_tag_full_list is None:
```

Comment:
Hmm None !=[]& hence this issue https://github.com/Azure/azure-cli/issues/18725 - Kota is working on the fix already. Let me know if I am missing something else here?

Reply:
Well, if you pay close attention to the result payload from the service tag API, the empty list is actually in a property called value - and not value**s** - hence the None instead of null...